package.json

```diff
@@ -52,6 +52,7 @@
     "glsl-noise": "^0.0.0",
     "lodash.omit": "^4.5.0",
     "lodash.pick": "^4.4.0",
+    "open-simplex-noise": "^2.4.0",
```

Comment:
I've been thinking about this, can we not use glsl-noise for this? I'd personally like to avoid adding dep's where we can...?

Reply:
I saw that glsl-noise was included in the deps, and was actually wondering that myself. I think it should be possible, I'll play around with it and see.

Reply:
@joshuaellis I think that glsl-noise package something specific to shaders? I only see it used in a .glsl file.

Reply:
That would make sense given the repo name. Have you looked at `simplex-noise`? Looking the repo for `open-simplex-noise` and reading more about it? It seems that the non-open version is faster? Thoughts?

Reply:
I'm open to either. I don't know how relevant it is (or if its even true) but I thought there is a patent on `simplex-noise`? `open-simplex-noise` seems to be maintained still where the simplex-noise package hasn't been updated in a while - maybe there is just no need to. No strong opinions either way, I'll try them both out.